Buildings/Resources/Data/Fluid/ZoneEquipment/FanCoilUnit/FanCoilAutoSize_ConstantFanVariableFlow.idf

```diff
@@ -0,0 +1,2121 @@
+!-Generator IDFEditor 1.51
```

Comment:
Has the files in package `Resources/Data/Fluid/ZonEquipment` been used any where?

I don't see the folder `Buildings.Fluid.ZoneEquipment`.